.github/ISSUE_TEMPLATE/research.md

```diff
@@ -0,0 +1,22 @@
+---
+name: 🍎 Research
```

Comment:
```suggestion
name: 🎓 Research
```

Reply:
The mortarboard is used in graduations, which are a celebration of academic success. I don't think this is appropriate, as this implies the research __must__ be successful, which is not the case.